### ISSUE TITLE
Bring back the horizontal padding option

### ISSUE THREAD

**Issue by ryu-ketsueki** [kupiqu/SierraBreezeEnhanced#68]
This started as a SierraBreeze fork, right? I've used SierraBreezeEnhanced in the past and it had this option called Horizontal Padding, which allowed to set a distance between the buttons and the edge of the titlebar. Without it, SierraBreezeEnhanced look without polish if its intention is to mimmick macOS, which does have the correct padding between the buttons and the edge of the titlebar.

**Comment by kupiqu** [comment]
> This started as a SierraBreeze fork, right?

![image](https://user-images.githubusercontent.com/3875196/75354719-7677b600-587b-11ea-924e-0f78f39761c3.png)

> Without it, SierraBreezeEnhanced look without polish if its intention is to mimmick macOS

That was not the intention, macOS style is a relatively recent add.

At some point I combined H & V padding because there were too many options. After that I removed font style, because there was no point, and independent padding could be recovered. However, I don't have time to do this myself. If someone was to submit a pull request, I will review it.
